frontend/src/components/object/ObjectList.vue

```diff
@@ -76,12 +76,7 @@ const closeUpload = () => {
 // };
 
 function onDeletedSuccess() {
-  toast.add({
-    severity: 'success',
-    summary: 'Success',
-    detail: 'File(s) deleted',
-    life: 3000
-  });
+  toast.success('Deleting file', 'File deleted');
```

Comment:
Style: Jarring presentation -> present-progressive and a past participle statement in one toast. Would recommend consulting with Tyler on this if this hasn't been raised.